src/os/ui/window/ogcserverhelp.js

```diff
@@ -0,0 +1,79 @@
+goog.module('os.ui.window.OgcServerHelpUI');
```

Comment:
This module should be in `src/plugin/ogc/ui`, and the view in `views/plugin/ogc/ui`.